VENTokenField/VENTokenField.m

```diff
@@ -375,19 +375,25 @@ - (UILabel *)toLabel
 
 - (void)adjustHeightForCurrentY:(CGFloat)currentY
 {
+    CGFloat oldHeight = self.height;
+    CGFloat height;
     if (currentY + [self heightForToken] > CGRectGetHeight(self.frame)) { // needs to grow
         if (currentY + [self heightForToken] <= self.maxHeight) {
-            [self setHeight:currentY + [self heightForToken] + self.verticalInset * 2];
+            height = currentY + [self heightForToken] + self.verticalInset * 2;
         } else {
-            [self setHeight:self.maxHeight];
+            height = self.maxHeight;
         }
     } else { // needs to shrink
         if (currentY + [self heightForToken] > self.originalHeight) {
-            [self setHeight:currentY + [self heightForToken] + self.verticalInset * 2];
+            height = currentY + [self heightForToken] + self.verticalInset * 2;
         } else {
-            [self setHeight:self.originalHeight];
+            height = self.originalHeight;
         }
     }
+    if (oldHeight != height
+        && [self.delegate respondsToSelector:@selector(tokenField:didChangeContentHeight:)]) {
+        [self.delegate tokenField:self didChangeContentHeight:height];
+    }
```

Comment:
thanks for that catch @ayanonagon 
done in a6c26aac8cf504ba56610558a370f5fa44eaa998